### DISCUSSION TITLE
Keymap to increase and decrease number under cursor

### DISCUSSION THREAD

**Discussion by matu3ba** [helix-editor/helix#675] [Q&A] [answered]
neovim has C-x and C-a for this, but C-x is already taken for completion.
What is the keymap in helix for this?

**Comment by pickfire** [comment]
C-x is taken for completion but that's in insert mode, so we can still use C-x and C-a in normal mode. It's not implemented yet.

**Comment by NNBnh** [comment, marked as answer]
Good news! <kbd>Ctrl</kbd> + <kbd>x</kbd> and <kbd>Ctrl</kbd> + <kbd>a</kbd> has been added to Helix:
- #967
- #1027

**Comment by m0n203** [comment]
when using vim, we can use `g c-a/x` to increase/decrease selected number list in visual mode, is that a way to archive same functionality in helix? how to generate increased number list in helix?
```
v_g_CTRL-A      g CTRL-A        2  add N to number in highlighted text                                                                                       
v_g_CTRL-X      g CTRL-X        2  subtract N from number in highlighted text
```

**Reply by gabydd** [reply]
Yes you can select the selections indices register with ``"#`` and do C-a or C-x

**Reply by m0n203** [reply]
it's works!
